src/celeritas/ext/detail/GeantBremsstrahlungProcess.hh

```diff
@@ -34,14 +34,14 @@ class GeantBremsstrahlungProcess : public G4VEnergyLossProcess
   public:
     // Construct with model selection
     explicit GeantBremsstrahlungProcess(ModelSelection selection);
-    // Empty destructor
-    ~GeantBremsstrahlungProcess();
 
-    // Prevent copying
-    GeantBremsstrahlungProcess&
-    operator=(GeantBremsstrahlungProcess const& right)
+    // Prevent copying and moving
+    GeantBremsstrahlungProcess& operator=(GeantBremsstrahlungProcess const&)
         = delete;
     GeantBremsstrahlungProcess(GeantBremsstrahlungProcess const&) = delete;
+    GeantBremsstrahlungProcess& operator=(GeantBremsstrahlungProcess&&)
+        = delete;
+    GeantBremsstrahlungProcess(GeantBremsstrahlungProcess&&) = delete;
```

Comment:
I don't understand why any of these have to be explicitly deleted? This is an implementation detail and it doesn't do any weird resource management.

Reply:
I don't think these were implicitly generated by the compiler since we declared a copy constructor/assignment. So it's just explicitly stating that these aren't available.

Reply:
I'm just saying I think the original explicit methods were superfluous and this should be "rule of zero"